runhouse/resources/hardware/cluster.py

```diff
@@ -54,7 +55,7 @@ def __init__(
         # Name will almost always be provided unless a "local" cluster is created
         name: Optional[str] = None,
         ips: List[str] = None,
-        ssh_creds: Dict = None,
+        creds: "Secret" = None,
```

Comment:
seems like this could also be a `str`? I see that in `def ssh_creds` you check if it could be a string and load secret if so

Reply:
or is that only if it is constructed via `from_config`? fine if so